app/org/maproulette/controllers/api/TaskController.scala

```diff
@@ -505,9 +505,15 @@ class TaskController @Inject() (
           .toMap
       )
 
+      val allReviewers = new scala.collection.mutable.ListBuffer[Long]()
+      tasks.foreach(t => {
```

Comment:
So a more scala idiosyncratic way of doing this would be to make it not mutable and use a flatMap
```
val reviewers = tasks.flatMap(t => {
    List(t.pointReview.reviewedBy.map(r => r)).flatMap(r => r) ++ t.pointReview.additionalReviewers.getOrElse(List())
})
```

Reply:
Thanks! I changed it in both places.